benefits/core/templates/core/includes/form.html

```diff
@@ -94,6 +92,8 @@
       {% endcomment %}
```

Comment:
@angela-tran Would it make sense to remove the recaptcha code from `base.html` and put it in this `if request.recaptcha` block in `form.html` instead?

```
    {% if request.recaptcha %}
      <script nonce="{{ request.csp_nonce }}" src="{{ request.recaptcha.script_api }}"></script>
    {% endif %}
```

I could do it in this PR.

Reply:
@machikoyasuda I am planning to do that as a PR for #2493. Will have it up soon

I think this PR has a good number of changes in it to test and review already, so probably best to not lump in the reCAPTCHA includes refactor. Even if it turns out to be a small code change, it feels like a separate PR to me.

Reply:
Good idea! Thank you :)